.github/CODEOWNERS

```diff
@@ -139,7 +139,8 @@ core/scripts/gateway @smartcontractkit/dev-services
 /integration-tests/**/*ccip* @smartcontractkit/ccip-offchain @smartcontractkit/core
 
 # Deployment tooling
-/deployment @smartcontractkit/ccip @smartcontractkit/keystone @smartcontractkit/core @smartcontractkit/deployment-automation
+/deployment/** @smartcontractkit/core @smartcontractkit/deployment-automation
+/deployment @smartcontractkit/ccip @smartcontractkit/keystone @smartcontractkit/core
 /deployment/ccip @smartcontractkit/ccip @smartcontractkit/core
```

Comment:
Adding /deployment/** won't help, because later /deployment/ccip overrides it. Codeowners aren't hierarchical in this sense, it's last-match-wins. You need, instead, to add deployment-automation to each of those product-specific lines.

Reply:
I see thanks for the pointer. fixed it here: https://github.com/smartcontractkit/chainlink/pull/15556/commits/01a1ac0b0c44ce3196228207708803d22ab5a711